src/styles/components/_usage-guidelines.scss

```diff
@@ -0,0 +1,54 @@
+$breakpoint-tablet: 1220px;
+
+.cmp-usage-guidelines {
+  background-color: #01f5ac;
+
+  &__heading {
+    position: relative;
+    font-family: Viga, sans-serif;
+    font-weight: 400;
+    font-size: map.get($font-sizes, "md");
+    padding-left: 2.5rem;
+  }
+
+  &__content {
+    padding: 0 1rem 6rem 3rem;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 70%;
+  }
+}
+
+@media only screen and (min-width: $breakpoint-tablet) {
+  .cmp-usage-guidelines {
+    background-color: #01f5ac;
+    background-image: url("../../../public/usage-guidelines-image.svg");
+    background-repeat: no-repeat;
+    background-position: top 50% right 10%;
+    background-size: 20rem;
+    width: 100%;
+    padding-left: 4rem;
+
+    &__content {
+      max-width: 70%;
+    }
+
+    &__heading {
+      position: relative;
+      font-family: Viga, sans-serif;
+      font-weight: 400;
+      font-size: map.get($font-sizes, "md");
+
+      // left align title
+      padding: 6rem 0 0 3rem;
```

Comment:
Setting this here is redundant after adding @bsmrdel101's suggestion above to add padding to the heading.

Reply:
I was just asking the apprentices about the CI check issue. I will do what you suggest! :)